en/Ciphers/rsa_cipher.md

```diff
@@ -0,0 +1,162 @@
+# RSA Cryptography Algorithm
+
+## About RSA Algorithm
+RSA (Rivest–Shamir–Adleman) is one of the first public-key cryptosystems and is widely used for secure data transmission. In such a cryptosystem, the encryption key is public and distinct from the decryption key which is kept secret (private).
+
+## Algorithm
+
+1. Select 2 Prime Numbers - **p & q**
+2. Calculate `n` as $$n = p * q$$
+3. Calculate Euler's Totient Function of n, $$φ(n) = (p-1) * (q-1)$$
```

Comment:
Perhaps make it clear that this formula for Euler's Totient is only applicable **iff** `p` and `q` are prime. Also consider explaining (briefly) what Euler's Totient *is*.

Reply:
Done